src/pe_source/pe_scripts.py

```diff
@@ -0,0 +1,129 @@
+"""A tool for gathering pe source data.
+
+Usage:
+    pe-source DATA_SOURCE [--log-level=LEVEL] [--orgs=ORG_LIST] [--cybersix-methods=METHODS] [--soc_med_included]
```

Comment:
Is there a reason this file is named `pe_scripts.py` and not `pe_source.py`?  It just seems unnecessarily confusing to this outside observer, but if there's a good reason, I'm interested in hearing it.

Reply:
Good point. Changed it to pe_source here: [b23a404](https://github.com/cisagov/pe-source/pull/1/commits/b23a40403758aa79a078b491e0072cf2f175edcf)